modules/nodes-descheduler-installing.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-installing_{context}"]
+= Installing the descheduler
+
+// TODO:
+The descheduler is not available by default. To enable the descheduler, you must install the Descheduler Operator from OperatorHub. After the Descheduler Operator is installed, you can then configure the eviction strategies.
+
+.Prerequisites
+
+// TODO: must be cluster admin?
+* Cluster administrator privileges.
+* Access to the {product-title} web console.
+
+.Procedure
+
+// TODO: Update this procedure once the operator is available in the OperatorHub
+
+// TODO: do we need to create a namespace for it first?
```

Comment:
Maybe operatorhub install creates the namespace for us? @ingvagabund did all of those manifests move to the CSV?

Reply:
I had to create NS on my own. Not sure if operatorhub can do it.